### PR TITLE
DTRUNETWO-50/Added updatePayPalAccount() method for V4 PayPal Accounts-Node

### PR DESCRIPTION
**Pull request by skambar110**
DTRUNETWO-50/Added updatePayPalAccount() method for V4 PayPal Accounts-Node.

### CODE REVIEW

**Review by arao6 — APPROVED** [review]
Looks good to me. pls pull the latest changes from V4 for the Travis build issues

**Review by arao6 — CHANGES_REQUESTED** [review]
Please also add status transitions for - listPayPalAccountStatusTransitions, getPayPalAccountStatusTransition, deactivatePayPalAccount, activatePayPalAccount

**Comment by skambar110** [comment]
> Please pull the latest changes from V4 for the Travis build issues (after PR# 40 is merged)

Latest changes have been pulled and then added few more methods which were listed in the comment.

**Comment by skambar110** [comment]
> Please also add methods for - listPayPalAccountStatusTransitions, getPayPalAccountStatusTransition, deactivatePayPalAccount, activatePayPalAccount

Added above mentioned methods.

File: src/Hyperwallet.js [hunk 1]
```diff
@@ -1054,6 +1120,49 @@ export default class Hyperwallet {
...
+            throw new Error("userToken is required");
+        }
+        if (!payPalAccountToken) {
+            throw new Error("payPalAccountToken is required");
+        }
+        if (!statusTransitionToken) {
+            throw new Error("statusTransitionToken is required");
+        }
+        this.client.doGet(`users/${encodeURIComponent(userToken)}/paypal-accounts/${encodeURIComponent(payPalAccountToken)}/status-transitions/${encodeURIComponent(statusTransitionToken)}`,
+            {},
```
Comment (Rrathinasabapath) [inline src/Hyperwallet.js:1143]: Please make line 1143 and 1144 into one.

**Review by Rrathinasabapath — COMMENTED** [review]
Except for one formatting change all look fine.

**Review by Rrathinasabapath — CHANGES_REQUESTED** [review]
Pls make the minor formatting change and push the change for approval further.

**Review by arao6 — APPROVED** [review]

File: src/Hyperwallet.js (hunk 1 above)
Comment (skambar110) [reply src/Hyperwallet.js]: Done.

**Comment by skambar110** [comment]
> Pls make the minor formatting change and push the change for approval further.

Minor formatting change is done and I have pushed the change for approval.

**Review by Rrathinasabapath — APPROVED** [review]
Verified. Ready to be reviewed by Vadim.

**Review by vkorotchenko — APPROVED** [review]

**Review by peter-joseph — APPROVED** [review]

**Pull request merged**